src/VSCodeExtension/syntaxes/qsharp.tmLanguage.json.v.template

```diff
@@ -46,11 +46,11 @@
       "patterns": [
         {
           "name": "keyword.control.qsharp",
-          "match": "\\b(if|elif|else|repeat|until|fixup|for|while|in|\\.\\.|w\/|return|fail)\\b"
+          "match": "\\b(using|borrowing|mutable|let|set|if|elif|else|repeat|until|fixup|for|in|while|return|fail|within|apply)\\b"
```

Comment:
We'll likely need to update the IQ# and LaTeX syntax highlighting as well.

Reply:
There are separate IQ# highlights? Should I file an issue for that?